DietPi-Software | Update Subsonic binary to v6.1.5

spectrumcomputing opened the pull request:
Update Subsonic to 6.1.5

Hopefully I did this right, its my first Pull Request.

spectrumcomputing committed:
Update dietpi-software

Update Subsonic to 6.1.5

MichaIng commented:
@spectrumcomputing 
Thanks for this.

Of course we need to place the binary onto our server (link target) first 😉. I am currently downloading it from: http://www.subsonic.org/pages/download.jsp

I will also adjust it to not contain the version info at all. So we can update the package without the need to touch `dietpi-software` code anymore.
_____
Changelog: https://github.com/MichaIng/DietPi/commit/c45d05ffb9b25a72140203f5d8b64ab30e31cc41

MichaIng committed:
v6.23

+ DietPi-Software | Subsonic: Our package does not contain the version string anymore, to allow updating for all DietPi versions without touching the code.

spectrumcomputing commented:
Thanks for doing this. In Subsonic the default music folder has gone back to what I assume is the install default of /var/music.  It worked fine when I changed it to /mnt/dietpi_userdata/Music
